pom.xml

```diff
@@ -23,6 +23,11 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+		<dependency>
+    <groupId>org.springframework.boot</groupId>
```

Comment:
You should be careful for code style in your code. You can select all code and then Ctrl + Shift + F click code to set code style. You should install to Google style in your ide.

Reply:
I installed. Thanks